src/test/java/org/edumips64/EndToEndTests.java

```diff
@@ -421,6 +421,83 @@ public void testSllv() throws Exception {
     runMipsTest("sllv.s");
   }
 
+  /* Test for instruction DDIVU */
+  @Test(timeout=2000)
+  public void testDdivu() throws Exception {
+    runMipsTest("ddivu.s");
+  }
+
+  /* Test for instruction DMULT */
+  @Test(timeout=2000)
+  public void testDmult() throws Exception {
+    runMipsTest("dmult.s");
+  }
+
+  /* Test for instruction DMULTU */
+  @Test(timeout=2000)
+  public void testDmultu() throws Exception {
+    runMipsTest("dmultu.s");
+  }
+
+  /* Test for instruction DSLLV */
+  @Test(timeout=2000)
+  public void testDsllv() throws Exception {
+    runMipsTest("dsllv.s");
+  }
+
+  /* Test for instruction DSRA */
+  @Test(timeout=2000)
+  public void testDsra() throws Exception {
+    runMipsTest("dsra.s");
+  }
+
+  /* Test for instruction DSRLV */
+  @Test(timeout=2000)
+  public void testDsrlv() throws Exception {
+    runMipsTest("dsrlv.s");
+  }
+
+  /* Test for instruction MULT */
+  @Test(timeout=2000000)
+  public void testMult() throws Exception {
+    runMipsTest("mult.s");
+  }
+
+  /* Test for instruction MULTU */
+  @Test(timeout=2000000)
+  public void testMultu() throws Exception {
+    runMipsTest("multu.s");
+  }
+  
+  /* Test for instruction ORI */
+  @Test(timeout=2000)
+  public void testOri() throws Exception {
+    runMipsTest("ori.s");
+  }
+
+  /* Test for instruction LUI */
+  @Test(timeout=2000000000)
```

Comment:
Why is this timeout so high?

Reply:
I was doing debugging, so I increased the timeout. I changed it back to 2000.